src/latency.h

```diff
@@ -89,4 +89,21 @@ void latencyAddSample(const char *event, mstime_t latency);
 #define latencyRemoveNestedEvent(event_var,nested_var) \
     event_var += nested_var;
 
+typedef struct durationStats {
+    unsigned long long cnt;
+    long long sum;
+} durationStats;
+
+typedef enum {
+    EL_DURATION_TYPE_EL = 0,
+    EL_DURATION_TYPE_CMD,
+    EL_DURATION_TYPE_IO_READ,
+    EL_DURATION_TYPE_IO_WRITE,
+    EL_DURATION_TYPE_AOF,
+    EL_DURATION_TYPE_CRON,
+    EL_DURATION_TYPE_NUM
```

Comment:
Put a comment on this line indicating that it must remain the last entry in the enum.